filters/encrypt/filter_operation.go

```diff
@@ -0,0 +1,33 @@
+package encrypt
+
+import "strings"
+
+// FilterOperation defines a type for filtering operations like: redact,
+// encrypt, hmac-sha256, etc.  Used in combination with a DataClassification, it
+// allows developers to override the default filter operations on tag fields.
+
```

Comment:
not a nit: godoc will not attach this comment to the type with the white space
```suggestion
```
<img width="1423" alt="Screen Shot 2021-06-25 at 12 02 30 PM" src="https://user-images.githubusercontent.com/6704214/123473296-3a5a7f80-d5ad-11eb-8066-60c22269d763.png">

Reply:
ty!